Dockerfile

```diff
@@ -0,0 +1,49 @@
+# This file is part of the Kimai time-tracking app.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+ 
+FROM php:7.2.9-apache-stretch
+
+WORKDIR /opt/kimai
+
+RUN php -r "readfile('http://getcomposer.org/installer');" | php -- --install-dir=/usr/bin/ --filename=composer && \
+    apt update && \
+    apt install -y \
+        git \
+        libicu-dev \
+        libjpeg-dev \
+        libldap2-dev \
+        libldb-dev \
+        libpng-dev \
+        unzip \
+        zip \
+        && \
+    docker-php-ext-install \
+        gd \
+        intl \
+        ldap \
+        pdo_mysql \
+        zip \
+        && \
+    docker-php-ext-install \
```

Comment:
I haven't setup a docker machine in a while and haven't seen these commands as well, so this question might be stupid... : exactly the same command is executed before (line 22), why are these commands duplicated here?

Reply:
That's a legacy from an (old) alpine based image I used for a drupal 8 project.  The was a bug where the php extensions needed to build twice.  Not only has it been fixed in the fpm image we have swapped to debian/apache.  I've cleaned it up.

Reply:
👍